src/EditorFeatures/CSharpTest/QuickInfo/SemanticQuickInfoSourceTests.cs

```diff
@@ -2471,7 +2471,7 @@ static void Main()
     {
         int[] a = n$$ew int[0];
     }
-}");
+}", MainDescription("int[]"));
```

Comment:
i relaxed quick info so that it can show the type of something if you hover over it.  I see nothing really bad about doing that, and it makes a lot of code simpler.